src/collectd.conf.pod

```diff
@@ -8778,6 +8778,14 @@ each metric being sent out.
 
 You can add multiple B<Attribute>.
 
+=item B<TTL> I<Int>
+
+Only available for KAIROSDB output format.
```

Comment:
Hi, actually I fixed them both in one commit https://github.com/collectd/collectd/pull/2252/commits/4381f2ebb2edafb4b17aac9d1fc78a57f03b6913
Sorry if it's not a best pratice.